lib/apipie/generator/swagger/method_description/api_schema_service.rb

```diff
@@ -47,9 +47,12 @@ def summary(api)
   end
 
   def tags
-    [@method_description.resource._id] +
-      warning_tags +
-      @method_description.tag_list.tags
+    tags = if Apipie.configuration.generator.swagger.skip_default_tags?
```

Comment:
Maybe it should be called `skip_default_resource_tag` since it still allows warning_tags? (Which has a separate config)

Reply:
If I'm not mistaken swagger tags are only used in endpoints so I guess it's ok to leave it as is, fine by me either way.